app/src/main/java/org/mifos/mobilebanking/ui/fragments/BeneficiaryApplicationFragment.java

```diff
@@ -312,7 +312,6 @@ public void showError(String msg ) {
         if (!Network.isConnected(getContext())) {
             sweetUIErrorHandler.showSweetNoInternetUI(nsvBeneficiary, layoutError);
         } else {
-            sweetUIErrorHandler.showSweetErrorUI(msg, nsvBeneficiary, layoutError);
```

Comment:
@therajanmaurya Fix 2: When Beneficiary was failed to create, then only Toaster should show, UI show stay the same.